### ISSUE TITLE
[Bug]: ADetailer produces higher brightness resulting in a visible square around faces

### ISSUE THREAD

**Issue by DarKxRaideR** [Bing-su/adetailer#209]
### Describe the bug

When I use ADetailer I get too bright results, it was working fine in the beginning but now it happens constantly. With a higher mask blur like 12 it becomes some sort of frame around the touched areas.
![00005-4004835305-portrait of some people are standing in front of a black wall  _lora_LowRA_1_-ad-before](https://github.com/Bing-su/adetailer/assets/125734866/ca8c6750-04d1-48d4-b301-52809624c128)
![00006-4004835305-portrait of some people are standing in front of a black wall](https://github.com/Bing-su/adetailer/assets/125734866/8859293e-749e-4a84-a9bc-3074a9afea5b)
![00012-4004835305-portrait of some people are standing in front of a black wall](https://github.com/Bing-su/adetailer/assets/125734866/690b21b9-82df-4c21-99dc-a87088eab71a)

### The entire console log, including python version information, webui version, commit hash, commandline arguments and error traces.

```Shell
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Version: v1.4.1-30-g46af86bd
Commit hash: 46af86bd3c7f0fc42cfea53a267c69e6cf03d87f
Installing requirements



Checking roop requirements
Install insightface==0.7.3
Installing sd-webui-roop requirement: insightface==0.7.3
Install onnx==1.14.0
Installing sd-webui-roop requirement: onnx==1.14.0
Install onnxruntime==1.15.0
Installing sd-webui-roop requirement: onnxruntime==1.15.0
Install opencv-python==4.7.0.72
Installing sd-webui-roop requirement: opencv-python==4.7.0.72


Launching Web UI with arguments: --xformers
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: C:\Users\Me\stable-diffusion-webui\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
[-] ADetailer initialized. version: 23.7.5, num models: 11
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
2023-07-15 11:13:49,059 - ControlNet - INFO - ControlNet v1.1.232
ControlNet preprocessor location: C:\Users\Me\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2023-07-15 11:13:49,111 - ControlNet - INFO - ControlNet v1.1.232
2023-07-15 11:13:50,896 - roop - INFO - roop v0.0.2
2023-07-15 11:13:50,897 - roop - INFO - roop v0.0.2
Loading weights [afcc6a9cac] from C:\Users\Me\stable-diffusion-webui\models\Stable-diffusion\realisticVisionV40_v40VAE.safetensors
Creating model from config: C:\Users\Me\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 9.3s (import torch: 1.2s, import gradio: 0.6s, import ldm: 0.4s, other imports: 0.6s, list SD models: 0.1s, load scripts: 5.1s, scripts before_ui_callback: 0.4s, create ui: 0.6s, gradio launch: 0.2s).
preload_extensions_git_metadata for 25 extensions took 1.41s
Applying attention optimization: xformers... done.
Textual inversion embeddings loaded(23): Asian-Less-Neg, bad-artist, bad-artist-anime, bad-hands-5, bad-image-9600, bad-image-v2-11000, bad-image-v2-27000, bad-image-v2-39000, bad_pictures, bad_prompt_version2, BadDream, badhandv4, CyberRealistic_Negative-neg, easynegative, EasyNegativeV2, FoundFootage, HyperStylizeV6, negative_hand-neg, ng_deepnegative_v1_75t, realisticvision-negative-embedding, rz-neg-15-foranalog, UnrealisticDream, verybadimagenegative_v1.3
Textual inversion embeddings skipped(5): nartfixer, nfixer, nrealfixer, rz-neg-foranalogportrait, rz-neg-general
Model loaded in 2.8s (load weights from disk: 0.4s, create model: 0.7s, apply weights to model: 0.6s, apply half(): 0.4s, move model to device: 0.6s).
100%|██████████████████████████████████████████████████████████████████████████████████| 29/29 [00:07<00:00,  3.69it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 29/29 [00:06<00:00,  3.80it/s]
0: 512x640 3 faces, 55.0ms
Speed: 1.0ms preprocess, 55.0ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)
100%|██████████████████████████████████████████████████████████████████████████████████| 12/12 [00:02<00:00,  5.22it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 12/12 [00:02<00:00,  4.26it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 12/12 [00:02<00:00,  4.27it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 29/29 [00:16<00:00,  1.72it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 29/29 [00:16<00:00,  3.80it/s]
```

### List of installed extensions

a1111-sd-webui-lycoris
adetailer
openpose-editor
sd-3dmodel-loader
sd-webui-additional-networks
sd-webui-aspect-ratio-helper
SD-WebUI-BatchCheckpointPrompt
sd-webui-controlnet
sd-webui-depth-lib
sd-webui-inpaint-anything
sd-webui-photopea-embed
sd-webui-roop
sd-webui-roop2
Stable-Diffusion-Webui-Civitai-Helper
stable-diffusion-webui-composable-lora
stable-diffusion-webui-rembg
stable-diffusion-webui-two-shot
ultimate-upscale-for-automatic1111

**Comment by Bing-su** [comment]
As far as I can see, this is not a bug. The way to fix it is to adjust the parameter and try some values that don't have this problem.

**Comment by DarKxRaideR** [comment]
As far as I can tell it seems to be related to the model I was using it with, after changing to another one it is no longer happening.

**Comment by QuietNoise** [comment]
Hope this helps someone with same issue.

I've found this problem quite pronounced on few SDXL models. Having said that, in all my cases the fix was to ensure that these models have an VAE set. Some examples:

**Without VAEs:**

![2023-12-01_19h33_22](https://github.com/Bing-su/adetailer/assets/3076187/f8aeb0f2-f699-4fc0-833e-6209429e28ce)
![2023-12-01_19h49_46](https://github.com/Bing-su/adetailer/assets/3076187/40cbab9b-c7e7-499a-a0d8-1ac14d81bfb5)
![2023-12-01_19h50_07](https://github.com/Bing-su/adetailer/assets/3076187/0dc59177-f335-4c2b-af98-8b7218c920d0)




**With VAE applied:**

![2023-12-01_19h41_42](https://github.com/Bing-su/adetailer/assets/3076187/ce15f5ea-4ff7-4076-9b4a-593e82328bd0)
![2023-12-01_19h42_07](https://github.com/Bing-su/adetailer/assets/3076187/a7794b7b-5ca2-4699-8681-de8e33026b78)
![2023-12-01_19h44_36](https://github.com/Bing-su/adetailer/assets/3076187/a042f67f-373a-4d98-ba75-a6fc0854e027)
